packages/salesforcedx-vscode-apex-debugger/package.json

```diff
@@ -54,7 +54,7 @@
     "postinstall": "node ./node_modules/vscode/bin/install",
     "test": "npm run test:vscode-integration",
     "test:vscode-integration":
-      "node ../../scripts/run-vscode-integration-tests-with-top-level-extensions",
+      "cross-env CODE_VERSION='1.30.0' node ../../scripts/run-vscode-integration-tests-with-top-level-extensions",
```

Comment:
Could this be added directly to the `run-vscode-integration-tests-with-top-level-extensions` script? I also noticed that isn't being added in the `salesforcedx-vscode-visualforce` package. Does that package need to be treated differently?

Reply:
We could move it there, I just didn't cause I focused only on the test suites that were failing. I think the `salesforcedx-vscode-visualforce` ones are still passing on version 1.31.0 because they do not rely on running a command via the cli, so the terminal changes do not affect them or the new info message is not messing with them.

Reply:
@allileong I ended up following your suggestion to fix appveyor timeouts, thanks :)